src/arbitraries/index.ts

```diff
@@ -21,7 +21,7 @@ export const real = (min = Number.MIN_SAFE_INTEGER, max = Number.MAX_SAFE_INTEGE
   min > max ? NoArbitrary : (min === max ? new ArbitraryConstant(min) : new ArbitraryReal(min, max))
 
 export const nat = (min = 0, max = Number.MAX_SAFE_INTEGER): Arbitrary<number> =>
-  new ArbitraryInteger(min, max)
+  max < 0 ? NoArbitrary : new ArbitraryInteger(Math.max(0, min), max)
```

Comment:
I agree with @RuiGuedes that this could be a `max < 0 ? NoArbitrary : integer(Math.max(0, min), max)`

Reply:
@hugoferreira Done!